ui/design-system-core/src/components/Anchor/__tests__/index.test.tsx

```diff
@@ -0,0 +1,57 @@
+import * as React from 'react';
+import { act, cleanup, fireEvent, getByTestId } from '@testing-library/react';
+import Anchor from '../';
+import { customRender } from '../../../test-utils';
+
+describe('<Anchor /> Component', () => {
+  let componentWrapper = customRender(<></>, {});
+
+  const link = 'https://akasha.world';
+  const testId = 'anchor-element';
+
+  const dummyClickFn = jest.fn(() => {
+    /** */
+  });
+
+  beforeEach(() => {
+    act(() => {
+      componentWrapper = customRender(<Anchor href={link} dataTestId={testId} />, {});
+    });
+  });
+
+  afterEach(() => {
+    act(() => componentWrapper.unmount());
+    cleanup();
+    jest.clearAllMocks();
+  });
+
+  it('renders correctly', () => {
+    expect(componentWrapper).toBeDefined();
+  });
+
+  it('has correct link', () => {
+    const { container } = componentWrapper;
+
+    const anchor = getByTestId(container, testId);
```

Comment:
It's recommended that we prioritise using getByRole query as it  best reflects how end users will interact with this the component. You can read about the query priorities [here](https://testing-library.com/docs/queries/about/#priority).